README.md

```diff
@@ -40,14 +40,16 @@ Series of steps to setup development environment
 * You will need to add the following to your `.env` file:
 ```
 NODE_ENV=development
-API_URL=http://wire.andela.com:8080/api
-ANDELA_API_BASE_URL=https://api-prod.andela.com
+API_URL=http://localhost:3000/api
+ANDELA_API_BASE_URL=https://api-staging.andela.com
```

Comment:
Any reason for the change here from prod to staging?

Reply:
- For `Local Development`, its better to make changes to the `staging API`.

Reply:
Great if there are instances we are making changes to the Andela API, that makes sense. I thought we were just consuming from the Andela API in which case prod would be the best env to consume from.

Reply:
@brandeddavid 
- I see your point, for the sake of `consuming` prod data and not `making changes`, i will revert to  `prod`.